File: ApiSearchGoogle/Models/SearchResult.cs
namespace ApiSearchGoogle.Models
{
    public class SearchResult
    {
        private string _titulo;
        private string _link;

        public SearchResult(string titulo, string link)
        {
            _titulo = titulo;
            _link = link;
        }

        public string Titulo 
        {
            get { return _titulo; }
        }

        public string Link 
        { 
            get { return _link; } 
            
        }
    }
}

File: AppSearchGoogle/src/app/models/SearchResult.ts
export class SearchResult {
    private _titulo: string;
    private _link: string;

    constructor(titulo: string, link: string) {
        this._titulo = titulo;
        this._link = link;
    }

    get titulo() {
        return this._titulo;
    }

    get link() {
        return this._link;
    }
}